libs/shared/left-side-menu/src/lib/shared-left-side-menu.scss

```diff
@@ -1,4 +1,9 @@
 $hover-color: #2196f3;
+$background: #ffffff;
+
+.MuiDrawer-paper {
```

Comment:
don't rewrite material ui classes, you should set your own style or class in editable component